main.js

```diff
@@ -0,0 +1,225 @@
+// 1
+
+function isAlarmSet(isEmployed, isOnVacation) {
+    return isEmployed && !isOnVacation;
+}
+
+// 2
+
+function isOneOrBothTruthy(valueOne, valueTwo) {
+    if (valueOne) {
+        return true;
+    }
+    if (valueTwo) {
+        return true;
+    }
+    return false;
+}
+
+function isOneTruthy(valueOne, valueTwo) {
+    if (valueOne && valueTwo) {
+        return false;
+    }
+    if (valueOne) {
+        return true;
+    }
+    if (valueTwo) {
+        return true;
+    }
+    return false;
+}
+
+// 3
+
+function compareDistanceToMargin(objectA, objectB, margin = 0) {
+    if (Math.abs(objectA - objectB) <= margin) {
```

Comment:
Declare a variable and give it the `Math.abs(objectA - objectB)` value. Make sure to choose a descriptive variable name.